client/src/Service/OrgService.php

```diff
@@ -224,6 +222,12 @@ protected function processChunks($chunks)
 
             $this->storeChunkOutput($upload);
             $this->logProgress($index + 1, $chunkCount);
+
+            if (!$logged) {
```

Comment:
May be missing something here but is `$logged` always false? I don't see any assignment happening in this class.